pkgdb2/ui/packages.py

```diff
@@ -1130,14 +1130,24 @@ def package_request_branch(package, full=True):
 def package_request_new():
     ''' Page to request a new package. '''
 
-    collections = pkgdb2.lib.search_collection(
-        SESSION, '*', 'Under Development')
-    collections.extend(pkgdb2.lib.search_collection(SESSION, '*', 'Active'))
-    pkg_status = pkgdb2.lib.get_status(SESSION, 'pkg_status')['pkg_status']
+    collections = pkgdb2.lib.search_collection(SESSION, '*', 'Under Development')
+    collections.reverse()
+    active_collections = pkgdb2.lib.search_collection(SESSION, '*', 'Active')
+    active_collections.reverse()
+    # We want all the branch `Under Development` as well as all the `Active`
```

Comment:
Actually this list is quite maintained as Under Development and active branches have different behaviors.

And technically, when Fn+1 is published, Fn-1 is still active for a month :)
So we would have to introduce a new status (pre-EOL?)